### PR TITLE
Removed duplicate menu button in navbar header.

### PR DESCRIPTION
**Pull request by membrq**
**Checklist**

- [x] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
- [x] My branch is up-to-date with the upstream `master` branch.
- [x] The tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works(if appropriate).
- [ ] I have added necessary documentation (if appropriate).

**Short description of what this resolves/which [issues](https://github.com/openSUSE/osem/issues) does this fix?:**

- #1910 Extra menu button in navbar when browsing website on mobile (on user side)

**Changes proposed in this pull request:**

- Removed duplicate menu button. 
- Added HTML style tag to float button to the right in the navbar header.

### CODE REVIEW

**Comment by differentreality** [comment]
@membrq I think you have misunderstood the issue. 

The left button (hamburger menu) is meant to be there for the menu in the admin side (there is a sidebar menu on the left side, which disappears on mobile devices, and becomes the second hamburger menu). 

What you have currently done is:
1) Remove the hamburger menu on the right, which was the user's drop down menu
2) For the splashpage specifically, you have left 1 hamburger menu on the right side (supposedly the users drop down menu) which in the code shows the admin sidebar (which shows nothing outside of the admin)

Please check again your changes locally before committing.

What we need to have is

Outside admin namespace:
1 hamburger menu on the right at all times, which is the user's drop down menu

Inside admin namespace:
Everything from before, plus one hamburger menu on the left side for the admin sidebar

Basically we do *not* want the left hamburger, which was the admin sidebar, appearing in the splashpage (splasphage is outside the admin area)

Does this make sense?

**Comment by membrq** [comment]
Ohh! Yes, that makes sense. I apologize for the misunderstanding! I'll rework my changes.

**Comment by bear454** [comment]
I did this a little differently for LFNW, where I wanted to keep the menu in the admin area, but get rid of that 2nd menu icon elsewhere:

https://github.com/bear454/osem/commit/906a8698e4e19b8de590b835bddbdd36f0fbffd0

**Comment by bear454** [comment]
Superceded by #2068 .
